src/app/shared/entity/backup.ts

```diff
@@ -94,8 +94,10 @@ type BackupStatusPhase = string;
 export const BackupStatusPhaseCompleted = 'Completed';
 
 export class EtcdRestore {
+  // ObjectMeta
```

Comment:
Let's remove it or if you prefer not to let's make it more clear that it's for the timestamps and the name.